src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
 import { connect } from "react-redux";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import {
+  AppstoreFilled,
+  ContainerOutlined,
+  LineChartOutlined,
+  SearchOutlined,
+} from "@ant-design/icons";
 import {
   AppContainer,
   Container,
   StyledLink,
   StyledList,
-  StyledNav,
-} from "./App.styles";
+  NavIcon,
+  NavText,
+  NarrowNav,
+  WideNav,
+  WideNavContainer,
+} from "styled";
```

Comment:
from  styled ?

https://media.giphy.com/media/KGSxFwJJHQPsKzzFba/giphy.gif?cid=ecf05e478lmruftv4wa9a9e7lu44wzm3fcttno1rhmfpe5kq&rid=giphy.gif&ct=g

Reply:
Yeah, that's how I named the folder with all the styled components. Should I change the name to something like "styles" ?